TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/redCloseAuto.java

```diff
@@ -50,34 +49,65 @@ public void driveToSpike(Drivebase driveBase, CubeSide side) {
         switch (side) {
             //if the left side is seen by the pipeline
             case Left:
-                driveBase.turnAngle(5.0, 0.5, telemetry);
-                driveBase.driveForward(31.5, 0.5, telemetry);
+                driveBase.turnAngle(5.0, 0.5, null);
+                driveBase.driveForward(31.5, 0.5, null);
             case Right:
-                driveBase.turnAngle(20.0, 0.5, telemetry);
-                driveBase.driveForward(29.625, 0.5, telemetry);
+                driveBase.turnAngle(20.0, 0.5, null);
+                driveBase.driveForward(29.625, 0.5, null);
             case Middle:
-                driveBase.turnAngle(10.0, 0.5, telemetry);
-                driveBase.driveForward(26.875, 0.5, telemetry);
+                driveBase.turnAngle(10.0, 0.5, null);
+                driveBase.driveForward(26.875, 0.5, null);
         }
     }
 
     /**
-     * Drive robot to the backboard using a path dependent on where the robot already is and the direction it's already facing.
+     * Drive robot to the backboard using a path dependent on where the robot already is and the direction it's already
+     * facing then park facing the back glass of the feild.
      * @param driveBase
      * @param side
      */
-    public void driveToBoard(Drivebase driveBase, CubeSide side)
+    public void driveToBackBoard(Drivebase driveBase, CubeSide side)
     {
         //if side is...
         switch (side){
             //if the left side is seen by the pipeline then the robot starts facing the 5 degree offset that it was before so...
             case Left:
-                
+            driveBase.turnAngle(85, 0.5, null);
+            driveBase.driveForward(57, 0.5, null);
+
```

Comment:
TurnAngle should be an absolute heading based off of the initial zero position from the robot start. If you wanted to turn to face the backboard, it would *always* be turnAngle (90, (speed), (telemetry));